### PR TITLE
ENG-1746 - wasm: HttpRequest changes

### PR DESCRIPTION
**Pull request by blinktag**
* Bump protos for all modules
* `httpRequest()` host function now receives a `WASMRequest` instead of `HttpRequestStep`

### CODE REVIEW

File: libs/wasm/detective/src/lib.rs [hunk 1]
```diff
@@ -50,7 +50,7 @@ pub extern "C" fn f(ptr: *mut u8, length: usize) -> u64 {
                         special_fields: Default::default(),
                     },
                 )),
-                special_fields: Default::default(),
+                ..Default::default()
```
Comment (blinktag) [inline libs/wasm/detective/src/lib.rs:53]: ISR now requires audience, but that is filled out by the SDK, so just set all defaults here

Comment (dselans) [reply libs/wasm/detective/src/lib.rs:53]: Let's add a ticket in backlog to remember to update all other SDKs to do it at some point.

Comment (dselans) [reply libs/wasm/detective/src/lib.rs:53]: ^ Can reference this issue.

**Review by dselans — APPROVED** [review]

File: libs/wasm/detective/src/lib.rs (hunk 1 above)
Comment (blinktag) [reply libs/wasm/detective/src/lib.rs:53]: Added ENG-1747 for other sdks

**Pull request merged**